sphinx_autotoc/__init__.py

```diff
@@ -166,23 +176,36 @@ def _add_to_nav(path: Path, docs: list[Path]) -> None:
             content = f.read()
 
     index_path = _get_dir_index(path)
-
-    pat = re.search(r"(\d\.\s)?(.*)", path.stem)  # 1. Text
-    if pat:
-        dirname_with_no_heading_nums = pat.group(2)
-    else:
-        return
+    dirname = trim_leading_numbers(path.name) if trim_folder_numbers else path.name
     search_paths = _make_search_paths(path, docs, False, True)
     with open(index_path.as_posix(), "w", encoding="utf-8") as f:
         f.write(
             NAV_PATTERN.format(
-                dirname=dirname_with_no_heading_nums,
+                dirname=dirname,
                 search_paths=search_paths,
                 includes=content,
             ).replace("\f", "\n   ")
         )
 
 
+def trim_leading_numbers(input: str) -> str:
+    """
+    Убирает из начала строки номер
+    1. Текст -> Текст
```

Comment:
```suggestion
>>> trim_leading_numbers("1. Текст")
'Текст'
```
Не уверен нужны ли кавычки в 'Текст'. Нужно проверить пройдет ли тест

Reply:
Отступы!

Reply:
Отступы сделал, про кавычки - не понял